RInAppMessaging/Classes/Repositories/CampaignRepository.swift

```diff
@@ -109,11 +112,22 @@ internal class CampaignRepository: CampaignRepositoryType {
         return updateImpressionsLeftInCampaign(campaign, newValue: campaign.impressionsLeft + 1)
     }
 
-    func loadCachedData() {
-        let cachedData = userDataCache.getUserData(identifiers: preferenceRepository.getUserIdentifiers())?.campaignData
-        campaigns.set(value: cachedData ?? [])
+    func loadCachedData(syncWithDefaultUserData: Bool) {
+        var cachedData = userDataCache.getUserData(identifiers: preferenceRepository.getUserIdentifiers())?.campaignData ?? []
```

Comment:
Is this the cached data for the newly logged in user?

Reply:
Might be. Or it can be also anonymous user cache. Depends on what's returned in `getUserIdentifiers()`